### PR TITLE
chore: clean-up ahead of some lint changes coming

### PR DESCRIPTION
**Pull request by devpow112**
Linting changes based on test work causes this to fail. Trying to minimize my PR on the testing swap to karma since it's quite large.

### CODE REVIEW

File: test/perceptual/sanity.visual-diff.js [hunk 1]
```diff
@@ -28,6 +25,8 @@ describe('sanity-test', () => {
 
 	it('first-test', async function() {
 		const rect = await visualDiff.getRect(page, '#test');
+
+		/* eslint-disable-next-line no-invalid-this */
```
Comment (MitchelPaulin) [inline test/perceptual/sanity.visual-diff.js:29]: In real tests there will be other uses of `this` which look invalid to the linter which are not. I think we should keep this at the top of the file with the intent to use this file as a template for other tests.

Another option would be to add a rule to ignore these file all together

Comment (devpow112) [reply test/perceptual/sanity.visual-diff.js:29]: I'll see if I can add a rule for it 👍

File: test/perceptual/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (devpow112) [inline test/perceptual/.eslintrc.json:1]: This will just blanket cover everything in `test/perceptual/`.

File: test/perceptual/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  "rules": {
+    "no-invalid-this": 0
```
Comment (MitchelPaulin) [inline test/perceptual/.eslintrc.json:3]:
```suggestion
    "no-invalid-this": "off"
```

Recommend using `off` for better readability

**Review by MitchelPaulin — APPROVED** [review]

**Pull request merged**